src/main/docs/guide/toc.yml

```diff
@@ -247,6 +247,10 @@ languageSupport:
   groovy: Micronaut for Groovy
   kotlin:
     title: Micronaut for Kotlin
+    kaptOrKsp: Kotlin support via KAPT or KSP
+    kapt: kapt compiler plugin
```

Comment:
Expand to proper titles:

```
    kapt: Kotlin Annotation Processing (KAPT)
    ksp: Kotlin Symbol Processing (KSP)
```

Reply:
close via https://github.com/micronaut-projects/micronaut-core/pull/8786/commits/3bcf6717a31b9a36e7012e9248d068842fa1abd3